packages/app/src/apis/vision.ts

```diff
@@ -3,9 +3,9 @@ import { dynamicModelExports } from './executable';
 
 /**
  * Classifies the image with given label, it returns the classification of this image.
- * @param img the input image type.
+ * @param _img the input image type.
  */
-export async function classify(img: Image): Promise<string> {
+export async function classify(_img: Image): Promise<string> {
```

Comment:
I think we can disable the unused check for this directory instead of renaming this argument.

Reply:
This names would be in our API reference, and those are weird IMHO :(